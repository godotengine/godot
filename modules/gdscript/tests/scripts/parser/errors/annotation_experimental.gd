#Â This annotation should be used within a documentation comment instead:
# ## @experimental: Reason here.

@experimental("This function isn't implemented yet.")
func say_hello():
    pass
