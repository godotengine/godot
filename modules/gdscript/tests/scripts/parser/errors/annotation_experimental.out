GDTEST_PARSER_ERROR
"@experimental" annotation does not exist. Use "## @experimental: Reason here." instead.
